lib/lightning_web/telemetry.ex

```diff
@@ -63,7 +63,19 @@ defmodule LightningWeb.Telemetry do
       summary("vm.memory.total", unit: {:byte, :kilobyte}),
       summary("vm.total_run_queue_lengths.total"),
       summary("vm.total_run_queue_lengths.cpu"),
-      summary("vm.total_run_queue_lengths.io")
+      summary("vm.total_run_queue_lengths.io"),
+
+      # Business metrics
+      summary("lightning.workorder.webhook.stop.duration",
+        tags: [:source_trigger_id],
+        unit: {:native, :millisecond},
+        description: "Time taken to trigger a new workorder"
+      ),
+      distribution("lightning.workorder.webhook.stop.duration",
```

Comment:
@taylordowns2000 `distribution` gives a breakdown of the distribution of the request durations, e.g. (x requests took n seconds, y requests took x + 1 seconds). I thought this might be useful.